### DISCUSSION TITLE
Domain transfer from Vercel to Cloudflare stuck

### DISCUSSION THREAD

**Discussion by vadimsdev** [vercel/community#7327] [Account Issue] [answered]
### Question

I want to transfer my domain vadims.dev from Vercel to Cloudflare. I have successfully initiated the transfer, but now Cloudflare shows the status of "Pending transfer" and the domain transfer page looks like this:

![image](https://github.com/vercel/community/assets/20427712/4371f398-9acb-4950-aeab-3f8f210cd5ed)

Could someone help me finalize the transfer process?

Thanks!

**Comment by jacobsamo** [comment, marked as answer]
When i transferred it took forever, not sure why but it eventually transferred. took over 5 days to completely transfer

**Reply by vadimsdev** [reply]
Thank you for the information, I initiated it on June 25th, so I hope it will be completed soon...

[a comment by an automated account is not shown]
